### PR TITLE
Update zenpack versions

### PR DESCRIPTION
**Pull request by tgjonestx**
Fixes ZEN-19517

PS. After this PR is merged, I plan on cherry-picking 17b72ed back to the develop branch

### CODE REVIEW

**Comment by jpeacock-zenoss** [comment]
Many of these platform zenpacks don't yet have a support/5.0.x branch.

File: listRepos.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```
Comment (jpeacock-zenoss) [inline listRepos.sh]: This should probably be `/bin/bash`.  On Ubuntu, `/bin/sh` is, unfortunately, no longer bash (Ubuntu decided to switch it to another shell called "dash").

**Comment by jpeacock-zenoss** [comment]
Just a note of interest, I also ran the listZenPacksVersions.sh script (after fixing the `#!` line) and found this in the output:

> ZenPacks.zenoss.vSphere/src/txcoprocess VERSION = read('.version')
> ZenPacks.zenoss.vSphere/src/txvsphere VERSION = read('.version')

I'm not sure if there's anything this script could usefully do to address this.

**Comment by tgjonestx** [comment]
RE: `= read('.version')` - yeah I don't think there's anything to do about in the script. The script is meant to  provide information.  It's up to the user to interpret and use the information appropriately. In this particular case, there is useful information - those particular zenpacks handle versioning a little differently from all of other zenpacks,

**Pull request merged**